packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart

```diff
@@ -43,19 +43,19 @@ class MethodChannelGoogleMapsFlutter extends GoogleMapsFlutterPlatform {
   /// This method is called when the plugin is first initialized.
   @override
   Future<void> init(int mapId) {
-    MethodChannel channel;
+    MethodChannel? channel;
     if (!_channels.containsKey(mapId)) {
       channel = MethodChannel('plugins.flutter.io/google_maps_$mapId');
       channel.setMethodCallHandler(
           (MethodCall call) => _handleMethodCall(call, mapId));
       _channels[mapId] = channel;
     }
-    return channel.invokeMethod<void>('map#waitForMap');
+    return channel!.invokeMethod<void>('map#waitForMap');
```

Comment:
This does not appear to be safe; the local variable is never initialized if _channels.containsKey(mapId) is true (which looks like a bug)